doc/classes/OptionButton.xml

```diff
@@ -139,7 +144,8 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
-				Select an item by index and make it the current item.
+				Selects an item by index and makes it the current item.
+				This will work even if the item is disabled.
```

Comment:
This could be put on the same paragraph as the line above (i.e. don't put a line break).